### ISSUE TITLE
Running on latest windows 11 against intune does not find firewall rules

### ISSUE THREAD

**Issue by celeroncool** [markstan/Test-IntuneFirewallRules#10]
Running .\Test-IntuneFirewallRules.ps1 on clean win11 machine against intune with admin rights outputs the following:

```
Type (Y)es to accept this warning and continue running this script:  y
Endpoint Security Policy: xxx- MDM Security Baseline
Endpoint Security Policy: xxx- Win10 Bitlocker
Endpoint Security Policy: xxx- Account protection
========================================================================================================================
Processing C:\Users\MDAdmin\AppData\Local\Temp\FWJSON22012025041013\xxx- Account protection_22-01-2025-16-10-21.817.json
========================================================================================================================

C:\Users\MDAdmin\AppData\Local\Temp\FWJSON22012025041013\xxx- Account protection_22-01-2025-16-10-21.817.json is not Firewall Rules template.  Skipping
========================================================================================================================
Processing C:\Users\MDAdmin\AppData\Local\Temp\FWJSON22012025041013\xxx- MDM Security Baseline_22-01-2025-16-10-19.670.json
========================================================================================================================

C:\Users\MDAdmin\AppData\Local\Temp\FWJSON22012025041013\xxx- MDM Security Baseline_22-01-2025-16-10-19.670.json is not Firewall Rules template.  Skipping
========================================================================================================================
Processing C:\Users\MDAdmin\AppData\Local\Temp\FWJSON22012025041013\xx- Win10 Bitlocker_22-01-2025-16-10-20.372.json
========================================================================================================================

C:\Users\MDAdmin\AppData\Local\Temp\FWJSON22012025041013\xxx- Win10 Bitlocker_22-01-2025-16-10-20.372.json is not Firewall Rules template.  Skipping
```
Customer name omitted with xxx.

None of those profiles are the firewall profile, but I can clearly see the profiles in intune:

![Image](https://github.com/user-attachments/assets/7c5dea18-b05b-415f-b534-963cc0d57959)
